src/routes.tsx

```diff
@@ -47,24 +50,25 @@ const routes = [
   {
     path: 'app',
     element: <DashboardLayout />,
-    children: routing,
+    children: routing
   },
   {
     path: 'test',
     element: <DashboardLayout />,
-    children: newRoutes,
+    children: newRoutes
   },
   {
     path: '/',
     element: <DashboardLayout />,
     children: [
-      { path: 'login', element: <LoginView /> },
-      { path: 'register', element: <RegisterView /> },
+      { path: 'login', element: <DefaultLoginView /> },
+      { path: 'resetpassword', element: <ResetPassword /> },
+      { path: 'signup', element: <SignupWithEmail /> },
       { path: '404', element: <NotFoundView /> },
-      { path: '/', element: <Navigate to="/app/home" /> },
-      { path: '*', element: <NotFoundView /> },
-    ],
-  },
+      { path: '/', element: <Navigate to="/app/dashboard" /> },
```

Comment:
tato jedna uprava co presne prosim robi? v demo.iteria.tech je uz zavedena home page

Reply:
vratil som to na /home